CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/SplitNodesPlugin.java

```diff
@@ -181,7 +181,12 @@ protected RecordStore query(final RecordStore query, final PluginInteraction int
         final RecordStore result = new GraphRecordStore();
 
         final Map<String, PluginParameter<?>> splitParameters = parameters.getParameters();
-        final String character = splitParameters.get(SPLIT_PARAMETER_ID) != null ? splitParameters.get(SPLIT_PARAMETER_ID).getStringValue() : "";
+        String character = "";
+        if (splitParameters.get(SPLIT_PARAMETER_ID) != null) {
+            if (splitParameters.get(SPLIT_PARAMETER_ID).getStringValue() != null) {
```

Comment:
I would just add this condition to the old tertiary operator.
i.e. `final String character = splitParameters.get(SPLIT_PARAMETER_ID) != null && splitParameters.get(SPLIT_PARAMETER_ID).getStringValue() != null ? ...`

Reply:
Yeah, that looks cleaner. Thanks for that @antares1470